src/main/groovy/com/smartbear/swagger/Swagger2TestCaseGenerator.groovy

```diff
@@ -45,13 +46,38 @@ class Swagger2TestCaseGenerator {
             it.name == parameter.name
         }
 
-        if (swaggerParam.default) {
-            parameter.setValue(String.valueOf(swaggerParam.default))
-        } else if (swaggerParam.example) {
-            parameter.setValue(String.valueOf(swaggerParam.example))
+        if (swaggerParam.required && StringUtils.isNullOrEmpty(parameter.value)) {
```

Comment:
Do we want to set values only for required parameters? I think we can still set the value if there is default or example value present for non-mandatory parameters. Generating the value makes sense for required parameters only.